p2p/protocol/identify/id.go

```diff
@@ -280,21 +314,14 @@ func (ids *IDService) broadcast(proto protocol.ID, payloadWriter func(s network.
 		go func(p peer.ID, conns []network.Conn) {
 			defer wg.Done()
 
-			// if we're in the process of identifying the connection, let's wait.
-			// we don't use ids.IdentifyWait() to avoid unnecessary channel creation.
```

Comment:
This is no longer a problem. If the connection has not been identified, we'll identify it. Otherwise, the channel will already exist.